{
  "id": "CVE-2021-3116",
  "tags": [
    {
      "label": "Product",
      "message": "n/a"
    },
    {
      "label": "Version",
      "message": "n/a"
    },
    {
      "label": "Vulnerability",
      "message": "n/a"
    }
  ],
  "description": " before_upstream_connection in AuthPlugin in http/proxy/auth.py in proxy.py before 2.3.1 accepts incorrect Proxy-Authorization header data because of a boolean confusion (and versus or).",
  "poc": {
    "references": [
      "https://cardaci.xyz/advisories/2021/01/10/proxy.py-2.3.0-broken-basic-authentication/"
    ]
  }
}
